src/disco/verify/fd_verify.c

```diff
@@ -39,7 +39,7 @@ during_frag( void * _ctx,
     return;
   }
 
-  if( FD_UNLIKELY( chunk<ctx->in[in_idx].chunk0 || chunk>=ctx->in[in_idx].wmark || sz > FD_TPU_DCACHE_MTU ) )
+  if( FD_UNLIKELY( chunk<ctx->in[in_idx].chunk0 || chunk>ctx->in[in_idx].wmark || sz > FD_TPU_DCACHE_MTU ) )
     FD_LOG_WARNING(( "chunk %lu %lu corrupt, not in range [%lu,%lu)", chunk, sz, ctx->in[in_idx].chunk0, ctx->in[in_idx].wmark ));
```

Comment:
```suggestion
    FD_LOG_WARNING(( "chunk %lu %lu corrupt, not in range [%lu,%lu]", chunk, sz, ctx->in[in_idx].chunk0, ctx->in[in_idx].wmark ));
```

Reply:
Why is this one just a warning?

Reply:
QUIC can overrun frags, which could cause a corrupt chunk if we are unlucky